Dockerfile

```diff
@@ -542,6 +556,15 @@ RUN \
   cmake -G"Unix Makefiles" -DCMAKE_INSTALL_LIBDIR=lib -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release .. && \
   make -j$(nproc) install
 
+RUN \
+  wget -O rubberband.tar.bz2 "$RUBBERBAND_URL" && \
+  echo "$RUBBERBAND_SHA256  rubberband.tar.bz2" | sha256sum --status -c - && \
+  tar xf rubberband.tar.bz2 && \
+  cd rubberband-* && \
+  meson -Dno_shared=true -Dfft=fftw build && \
+  ninja -j$(nproc) -vC build install && \
+  echo "Requires.private: fftw3 samplerate" >> /usr/local/lib/pkgconfig/rubberband.pc
```

Comment:
Wonder if this could be reported upstream?

Reply:
Seems meson pkgconfig has some kind of support for this https://mesonbuild.com/Pkgconfig-module.html but i haven't used it myself. Maybe should just file a issue about it?

Reply:
yes, i actually started reading about it a while back today.. i see the build file and somewhat understand the process but i personally can't justify the time to further investigate. Reporting is something to consider